Updated Starlette/FastAPI setup documentation (#5472)
getsentry/sentry-docs: src/wizard/python/fastapi.md

pip install --upgrade 'sentry-sdk[fastapi]'
```

> 2. To configure the SDK, initialize it before your app has been initialized:
> 
> Sentry SDK will auto enabled support for FastAPI if you have the `fastapi` Python package installed in your project. There is no configuration options you need to add when initializing the Sentry SDK, everything works out of the box.

Reviewer: ```suggestion
2. To configure the SDK, initialize it before your app has been initialized. The Sentry SDK automatically enables support for FastAPI if you have the `fastapi` Python package installed in your project. There are no configuration options you need to add when initializing the Sentry SDK as everything works out of the box:
```
